docs/404.md

```diff
@@ -0,0 +1,6 @@
+---
+permalink: /404.html
+---
+# Gateway API
+
+Our docs have moved to [gateway-api.sigs.k8s.io](https://gateway-api.sigs.k8s.io).
```

Comment:
Can/Should we auto-redirect?
https://superdevresources.com/redirects-jekyll-github-pages/

Reply:
I had no idea that was possible, but that's a great idea, thanks! Going to experiment with some of these.

Reply:
Good call, that is a lot better. Appears to be working well at https://robscott.github.io/gateway-api (GitHub pages on my fork is configured to use this branch).